cypress/support/util.js

```diff
@@ -502,4 +502,24 @@ export const createForm = (name,blankForm = true) => {
 export const saveFormAndReload = ()  => {
 	cy.get('.caldera-header-save-button').click();
 	cy.reload();
-};
\ No newline at end of file
+};
+
+/**
+ * Variables functions
+ * 
+ * @since 1.8.10
+ */
+export const cfGoToVariablesTab = () => {
+	cy.get('#tab_variables a').click();
+};
+export const cfAddVariable = () =>{
```

Comment:
@New0 missing inline docs. It was really helpful to me when I looked at these functions for first time in a year, that they had one line describing what they do.

Reply:
You're right I corrected that. I had it set like a header for a group of paragraphs.